build_library/generate_grub_hashes.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
```

Comment:
Yeah. It's somewhat difficult to mechanically generate the list of valid grub configuration parameters, so I think a validation run and test is going to be the best approach here.